build/common/installer/scripts/tomlparser.rb

```diff
@@ -260,6 +260,16 @@ def get_command_windows(env_variable_name, env_variable_value)
   file = File.open("setenv.ps1", "w")
 
   if !file.nil?
+    # This will be used in fluent-bit.conf file to filter out logs
+    if (!@collectStdoutLogs && !@collectStderrLogs)
```

Comment:
I presume, you have verified by setting both options to false and works as expected.

Reply:
checked and works as expected!